content/en/docs/v3.5/install.md

```diff
@@ -79,7 +79,7 @@ TBD---Help Wanted
 
 ## Installation on Kubernetes, using a statefulset or helm chart
 
-TBD---Help Wanted
+You can follow the instructions provided by [Bitnami's etcd Helm chart]
```

Comment:
Suggest a slight change to state why we are directing readers to the Bitnami chart. Something like:

> The etcd project does not currently maintain a helm chart, however you can follow the instructions provided by [Bitnami's etcd Helm chart].